### ISSUE TITLE
Command group translation migration

### ISSUE THREAD

**Issue by chennes** [FreeCAD/FreeCAD-translations#204]
According to @wwmayer [here](https://github.com/FreeCAD/FreeCAD/pull/9346#issuecomment-1518685471):

> Having the group name marked for translation has the side-effect that for every command the same text must be translated over and over again. To reduce this extra effort we once have decided to not mark the group name any more but instead add the line qApp->translate("CommandGroup", "View"); so that it must be translated only once.
> 
> So far this is only implemented for the core system and the Sketcher module.
> 
> So, not marking the group name for translation is correct and we should start to migrate all other modules, too.

That is, in code that formerly looked like this:
```
CmdPartMakeSolid::CmdPartMakeSolid()
  :Command("Part_MakeSolid")
{
    sAppModule    = "Part";
    sGroup        = QT_TR_NOOP("Part");
    sMenuText     = QT_TR_NOOP("Convert to solid");
    sToolTipText  = QT_TR_NOOP("Create solid from a shell or compound");
    sWhatsThis    = "Part_MakeSolid";
    sStatusTip    = sToolTipText;
    sPixmap       = "Part_MakeSolid";
}
```
We should eliminate the `QT_TR_NOOP` from the `sGroup` line.

**Comment by kaktusus** [comment]
Is this ticket can be closed?

**Comment by chennes** [comment]
No, there are still 114 matches on a search for "sGroup.*=.*QT_TR_NOOP".
